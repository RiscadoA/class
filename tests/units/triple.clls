proc main() {
    cut {
        println("close x");
        close x
        |x:wait|
        println("close y");
        close y
        |y:wait|
        println("wait y");
        wait y;
        println("wait x");
        wait x;
        println("end");
        ()
    }
};;
