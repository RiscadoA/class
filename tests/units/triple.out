close x
close y
wait y
wait x
end
